File: OIS/ois_mana/mana.pas
{ NOTE: it is recommended to use this even if you don't understand the following code }

const
    MAXN = 100000;

var
    K, N, i : LongInt;
    C       : Array[0..MAXN-1] of LongInt;
    M       : Int64;

begin
{
    uncomment the two following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    ReadLn(N, K);

    for i:=0 to N-1 do
        Read(C[i]);
    ReadLn();

    M := 0;

    { INSERT YOUR CODE HERE }


    WriteLn(M);

end.

File: OIS/ois_mana/mana.c
// NOTE: it is recommended to use this even if you don't understand the following code.

#include <assert.h>
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

#define MAXN 100000

int       K, N, i;
int       C[MAXN];
long long M;

int main() {
    // uncomment the two following lines if you want to read/write from files
    // freopen("input.txt", "r", stdin);
    // freopen("output.txt", "w", stdout);

    assert(2 == scanf("%d%d", &N, &K));

    for (i = 0; i < N; ++i)
        assert(1 == scanf("%d", &C[i]));

    M = 0;


    // INSERT YOUR CODE HERE


    printf("%lld\n", M);

    return 0;
}
